clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -1192,13 +1193,40 @@ llvm::Value *CodeGenFunction::EmitLoadOfCountedByField(
   return nullptr;
 }
 
+LangOptions::StrictFlexArraysLevelKind
+CodeGenFunction::effectiveArrayBoundsFlexArraysLevel() {
+  using StrictFlexArraysLevelKind = LangOptions::StrictFlexArraysLevelKind;
+  using ArrayBoundsStrictFlexArraysLevelKind =
+      LangOptions::ArrayBoundsStrictFlexArraysLevelKind;
+  StrictFlexArraysLevelKind StrictFlexArraysLevel;
+  switch (getLangOpts().getArrayBoundsStrictFlexArraysLevel()) {
+  case ArrayBoundsStrictFlexArraysLevelKind::Default:
+    StrictFlexArraysLevel = StrictFlexArraysLevelKind::Default;
```

Comment:
`return StrictFlexArraysLevelKind::Default;`

Reply:
https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code

Not about switch but I think it applies.